editor/src/components/canvas/canvas-strategies/strategies/shared-move-strategies-helpers.ts

```diff
@@ -170,6 +171,44 @@ export function applyMoveCommon(
   }
 }
 
+function getAppropriateLocalFrame(
+  options: MoveCommandsOptions | undefined,
```

Comment:
nitpick, but I'd rather just have this here: 
```suggestion
  options: { ignoreLocalFrame: boolean } | undefined,
```

Reply:
this was really just a nitpick, in general I like the idea that we rely on duck typing, to avoid turning things like MoveCommandsOptions into huge god-interfaces during a refactor.